src/v/schema/tests/fake_registry.cc

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2024 Redpanda Data, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+#include "schema/tests/fake_registry.h"
+
+#include <seastar/core/coroutine.hh>
+#include <seastar/core/future.hh>
+
+namespace {
+// NOLINTNEXTLINE(cppcoreguidelines-avoid-non-const-global-variables,cert-err58-cpp)
+static ss::logger dummy_logger("schema_test_logger");
+
+namespace ppsr = pandaproxy::schema_registry;
+
+} // namespace
+
+ss::future<ppsr::canonical_schema_definition>
```

Comment:
the commit message says we are moving code ("move wasm fake_schema_registry to schema::fake_registry") but i only see additions. did you mean to delete something?